news/5490.doc

```diff
@@ -0,0 +1 @@
+Change sentence from future to present tense
```

Comment:
I don't think this needs an entry in the NEWS file.

Could you please change this file's extension to `trivial`?